### DISCUSSION TITLE
Does it help with aim in general?

### DISCUSSION THREAD

**Discussion by luxzg** [Digitalroot-Valheim/Digitalroot.Valheim.SlopeCombatFix#2] [Q&A]
Eg. for removing stubborn stumps, or cutting logs? If yes, maybe add it to the description on Nexus/Thunder :)

**Comment by Digitalroot** [comment]
I am not sure. Maybe one of the long time users of the old version might be able to provide some insight.

**Comment by scorp10n2000** [comment]
I haven't tried this one yet but Slope Combat Fix mod does improve aim slightly for cutting logs etc especially at an angle like a slope.
